docker/pypi/wmagent-base/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.cern.ch/cmsweb/oracle:21_5-stable as oracle
+FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230525
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+# directory where we place all the relevant scripts and configurations for WMAgent docker image
+ENV WMA_ROOT_DIR=/data
+
+# Install basic OS package dependencies
+RUN apt-get update
+RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils cron mariadb-server myproxy voms-clients rlwrap libaio1 && apt-get clean
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps && apt-get clean
+
+# copy oracle client:
+COPY --from=oracle /usr/lib/oracle /usr/lib/oracle
+ENV LD_LIBRARY_PATH=/usr/lib/oracle
+ENV PATH=$PATH:/usr/lib/oracle
+ENV PKG_CONFIG_PATH=/usr/lib/oracle
+
+# Add all deployment needed directories, install and run scripts
+ADD etc $WMA_ROOT_DIR/etc
+ADD manage $WMA_ROOT_DIR/manage
+ADD install.sh $WMA_ROOT_DIR/install.sh
```

Comment:
Reading more about the it I do not think even the `install.sh` and `run.sh` script placement is good to be in the `wmagent-base` image, since they depend on variables and setup parameters that come later with the other image you suggest.